gcloud-java-logging/src/main/java/com/google/gcloud/logging/spi/v2/testing/LocalLoggingImpl.java

```diff
@@ -0,0 +1,97 @@
+package com.google.gcloud.pubsub.spi.v2.testing;
+
+import com.google.logging.v2.DeleteLogRequest;
+import com.google.logging.v2.ListLogEntriesRequest;
+import com.google.logging.v2.ListLogEntriesResponse;
+import com.google.logging.v2.ListMonitoredResourceDescriptorsRequest;
+import com.google.logging.v2.ListMonitoredResourceDescriptorsResponse;
+import com.google.logging.v2.LogEntry;
+import com.google.logging.v2.LoggingServiceV2Grpc;
+import com.google.logging.v2.ReadLogEntriesRequest;
+import com.google.logging.v2.ReadLogEntriesResponse;
+import com.google.logging.v2.WriteLogEntriesRequest;
+import com.google.logging.v2.WriteLogEntriesResponse;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.Empty;
+
+import io.grpc.stub.StreamObserver;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LocalLoggingImpl implements LoggingServiceV2Grpc.LoggingServiceV2 {
```

Comment:
I am not sure why we would want to make it public (and Helper could return the interface rather impl).